pkg/prow/pjutil/pjutil.go

```diff
@@ -85,10 +81,10 @@ func createRefs(pr *scm.PullRequest, baseSHA string) plumber.Refs {
 	}
 }
 
-// NewPresubmit converts a config.Presubmit into a builder.PlumberJob.
+// NewPresubmit converts a config.Presubmit into a builder.PlumberArguments.
 // The builder.Refs are configured correctly per the pr, baseSHA.
 // The eventGUID becomes a github.EventGUID label.
-func NewPresubmit(pr *scm.PullRequest, baseSHA string, job config.Presubmit, eventGUID string) plumber.PlumberJob {
+func NewPresubmit(pr *scm.PullRequest, baseSHA string, job config.Presubmit, eventGUID string) plumber.PlumberArguments {
```

Comment:
What is a `presubmit` in the context of `jenkins-x` pipeline.

Reply:
it a PR pipeline as opposed to a release pipeline - these names match the current prow config we use